zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/Notifier.java

```diff
@@ -0,0 +1,17 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import com.github.onsdigital.zebedee.model.Collection;
+
+
+/**
+ * Notifier is a generic interface for sending notifications to external systems such as Slack
+ */
+public interface Notifier {
+
+    /**
```

Comment:
Missing description in Javadoc. You've explained the params but not when the method does 😂